topics/intro/content_updates.md

```diff
@@ -12,6 +12,11 @@ See [GitHub Changelog](https://github.com/JetBrains/intellij-sdk-docs/commits/ma
 
 ## 2022
 
+### June-22
+
+Postfix Completion
+: Add [](postfix_completion.md) section explaining how to implement generating or wrapping the existing code into additional constructs without navigating the caret back.
```

Comment:
"section" could be replaced with "pages" in this case eventually

Reply:
Hmm, I think this is a section with two subpages and sounds better than "Postfix Completion (<- clickable link) pages". I would leave it as is.

Reply:
I think we should not overcomplicate things because last time I also spend 10min thinking whether to use "section", "item", "entry", etc.. It's just not worth the time for an update entry.

Reply:
Leaving it as is then, thanks.